lib/internal/assert/assertion_error.js

```diff
@@ -188,8 +186,8 @@ function createErrDiff(actual, expected, operator) {
 
   let printedLines = 0;
   let identical = 0;
-  const msg = kReadableOperator[operator] +
-        `\n${colors.green}+ actual${colors.white} ${colors.red}- expected${colors.white}`;
+  const error_msg = message != null ? message : kReadableOperator[operator];
```

Comment:
```suggestion
  const error_msg = message ?? kReadableOperator[operator];
```
also this should probably be in camelCase